language/German/strings.po

```diff
@@ -1880,7 +1880,7 @@ msgstr "Videomodus"
 
 #: src/frontend/mame/ui/submenu.cpp:102
 msgid "Number Of Screens"
-msgstr ""
+msgstr "Anzahl der Bildschirme"
 
 #: src/frontend/mame/ui/submenu.cpp:104
 msgid "Triple Buffering"
```

Comment:
"Dreifache Zwischenspeicherung" finde ich nicht so schön, bei "dreifacher Pufferung" schlägt mir Google automatisch "Dreifachpufferung" vor, was auch als Begriff in der Wikipedia steht, demnach vielleicht die übliche Übersetzung von Triple Buffering?